packages/charts/src/types.ts

```diff
@@ -176,6 +181,7 @@ export type Theme = {
   axisY?: Axis;
   gridX?: Grid;
   gridY?: Grid;
+  axisTitle: Typography;
```

Comment:
I believe it will be better to extend the `Axis` type with title property - instead of creating another abstraction.